apps/sync/js/sync-engine/syncengine.js

```diff
@@ -305,15 +312,13 @@ ting to fetch resource.`) {
       }).then(() => {
         return this._syncCollection('meta');
       }).then(() => {
-        return this._getItem('meta', 'global');
+        return this._getItem('meta', 'global', false);
```

Comment:
`return this._getItem('meta', 'global', false /* syncIfNeeded */);` or simply `return this._getItem('meta', 'global');`